src/trans/target.cpp

```diff
@@ -59,6 +59,13 @@ const TargetArch ARCH_POWERPC64LE = {
     { /*atomic(u8)=*/true, true, true, true,  true },
     TargetArch::Alignments(2, 4, 8, 16, 4, 8, 8)
 };
+// see https://webassembly.org/docs/portability/
+const TargetArch ARCH_WASM32 = {
+    "wasm32",
+    32, false,
+    { /*atomic(u8)=*/true, true, true, false,  true },
```

Comment:
Wasm32 supports 64bit atomics with the atomics extension. Without it doesn't support any atomics at all.

Reply:
@bjorn3 thanks